src/docs/deployment/android.md

```diff
@@ -317,7 +265,8 @@ The release bundle for your app is created at
 `<app dir>/build/app/outputs/bundle/release/app.aab`.
 
 By default, the app bundle contains your Dart code and the Flutter
-runtime compiled for [armeabi-v7a][] (32-bit) and [arm64-v8a][] (64-bit).
+runtime compiled for [armeabi-v7a][] (ARM 32-bit), [arm64-v8a][]
```

Comment:
Can you put in a comma at the end of the sentence? We like the Oxford comma.  :D